frontend/awx/Awx.tsx

```diff
@@ -1,10 +1,68 @@
+import '@patternfly/patternfly/patternfly-base.css';
+import '@patternfly/patternfly/patternfly-charts.css';
+
+import '@patternfly/patternfly/patternfly-charts-theme-dark.css';
+
+import '../common/styles.css';
+
 import { Page } from '@patternfly/react-core';
+import { Suspense } from 'react';
+import { createRoot } from 'react-dom/client';
+import { useTranslation } from 'react-i18next';
+import { BrowserRouter, Navigate, Route, Routes, useNavigate } from 'react-router-dom';
+import { PageFramework } from '../../framework';
+import ErrorBoundary from '../../framework/components/ErrorBoundary';
+import { Login } from '../common/Login';
 import { AnsibleMasthead } from '../common/Masthead';
+import { PageNotFound } from '../common/PageNotFound';
+import { RouteObj } from '../common/Routes';
+import '../common/i18n';
 import { ActiveUserProvider } from '../common/useActiveUser';
 import { AwxRouter } from './AwxRouter';
 import { AwxSidebar } from './AwxSidebar';
-import { WebSocketProvider } from './common/useAwxWebSocket';
 import { AwxConfigProvider } from './common/useAwxConfig';
+import { WebSocketProvider } from './common/useAwxWebSocket';
+
+const container = document.createElement('div');
+container.style.position = 'fixed';
+container.style.width = '100%';
+container.style.height = '100%';
+container.style.overflow = 'hidden';
```

Comment:
Why is this all necessary? Fixed positioning seems odd here.

I would suggest only min-height: 100% (or 100dvh)

Reply:
This is so that the main container is exactly the size of the window and has no overflow.
That is needed so that subsections of the page can handle the overflow without the whole pages getting a scrollbar.